js/views/settingsVw.js

```diff
@@ -758,6 +787,7 @@ module.exports = Backbone.View.extend({
     if(newAddress.name || newAddress.street || newAddress.city || newAddress.state || newAddress.postal_code) {
       if(!newAddress.name || !newAddress.street || !newAddress.city || !newAddress.state || !newAddress.postal_code){
         messageModal.show(window.polyglot.t('errorMessages.saveError'), window.polyglot.t('errorMessages.missingError'));
+        setTimeout(() => $(e.target).removeClass('loading'), 0);
```

Comment:
Not sure exactly, I'd have to think about it. To be honest, my first thought is that I find this approach of globally targeting and modifying an element like this really scary. I think a better approach would probably be to either make a Loading Button view or a jQuery plugin. Both would give you some encapsulation and allow you to add an API around this behavior.